input/fsh/examples/biospecimen.fsh

```diff
@@ -0,0 +1,31 @@
+/*Example for Sample*/
+
+Instance: SA-000 /*Collection Event ID can't have underscores*/
+InstanceOf: NCPICollectedSample
+Title: "Example biospecimen based on data from CBTN"
+Usage: #example
+Description: "Example biospecimen based on data from CBTN"
+* identifier.value = "SA_RV52EY7S" /*Sample ID*/
+* subject = Reference(PT_006SP660) /*Participant ID*/
+* type.text = "Peripheral Whole Blood" /*Sample Type*/
+/*There is a SNOMED term for peripheral blood specimen OR 
+https://github.com/include-dcc/include-model-forge/blob/main/input/fsh/codesystems/CodeSystem-SampleTypes.fsh 
+to access this codesystem for now but we defintely need a real ontology for describing things like blood draws etc.*/
+* collection.method.text = "Blood Draw" /*Biospecimen Method*/
+* collection.method.coding = $loinc#LP125037-4 /*Biospecimen Method*/
+* container.identifier.value = "BS_9QM8EXG3" /*Aliquot.AliquotID*/
+* collection.collectedDateTime.extension[+]
+  * url = $cqf-relativeDateTime
```

Comment:
This looks great and will be a helpful cheat sheet for the next time you have to use one of those CQF Extensions.